scss/plugins/angular-ui-bootstrap-reskin/angular-ui-bootstrap-reskin.scss

```diff
@@ -35,3 +35,5 @@ $vars: $pluginVars;
 @import "components/collapse";
 // Daterange
 @import "components/daterange";
+// Day Block
+@import "components/day-block";
```

Comment:
i'm not sure its place is with the plugin reskinning angular-ui-bootstrap directives